types/transaction.d.ts

```diff
@@ -42,7 +42,7 @@ export interface MinTransactionParams {
   code?: string;
   deployer?: string;
   params?: CallParam[];
-  unsafeParams?: Uint8Array;
+  unsafeParams?: { [key: number]: number };
```

Comment:
here no any resone use this intarface, it should be `Uint8Array`

Reply:
in RAM each element will store 128bit, it hould be 8bit

Reply:
If your intention was to clone the payload object you should use 
https://www.geeksforgeeks.org/lodash-_-clonedeep-method/ then all the types will be kept

Reply:
So there is 2 options to fix this. Either use cloneDeep and pass the payload containing the unsafeParams as an Uint8Array class instance. Or doing what i did

Reply:
in chrome is not possible to send object between context, so it anyway should be parsed in `Uint8Array` in the end, unsafe params possible to be bigger, so it can hit by preformance.
<img width="1085" alt="image" src="https://github.com/bearby-wallet/bearby-extension/assets/21293723/6057fe98-ee44-4423-bba9-e0dd7b973129">

by the image you can see thats parse as Uint8Array

Reply:
i cannot use lodash this is bad way import a library for one methom. Anyway you cannot send message between context

Reply:
not sure to understand your point. Do you see the bug ? i would be happy to fix it differently, but right now its not working.
If i understand well the payload must be in json stringified (i.e it can not contains a Uint8Array class instance).
if you want to avoid the `{ [key: number]: number };` we can transform it to `number[]` by changing this line:

https://github.com/bearby-wallet/bearby-web3/blob/fb706d04ee507768dc7b103a946e09fca02333c9/lib/transaction.ts#L36
to
`        unsafeParams: this.unsafeParams?.buffer,`
Doing this you take only the byte buffer instead of the Uint8Array instance. but the type will be `number[]` in bearby extension

Reply:
> by the image you can see thats parse as Uint8Array
This is not Uint8Array, its just an array of number ( number[]

Reply:
it an be base58 or hex for transfer between context, `number` type very bad to use it

Reply:
> > by the image you can see thats parse as Uint8Array
> 
> This is not Uint8Array, its just an array of number ( number[]) which is supported in json

yes, which after transfer to background page convert to Uint8Array.

Reply:
i can upgrade bearby-web3 after it will use hex or base58 for sending between messages

Reply:
> Yes fine! any solution is good to me as soon as it make the thing work :) feel free to drop this PR and implement another solution!

don't need drop this PR just change one thing )))

Reply:
Keep in mind that types are only used for build time. Here i am not changing the type of the data at runtime, i am just changing the type to it reflect the real type of the data in the current state of the code.

if we set Uint8Array here, it will work if we keep this `Uint8Array.from(Object.values(confirmParams.unsafeParams))`  but it will be misleading because its not the real type of the data at runtime. you can check yourself using "instanceof" on confirmParams.unsafeParams , you will see.